cinema/migrations/0003_alter_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-12-14 00:34
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0002_movie_image"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(
+                null=True, upload_to=cinema.models.movie_image_path

```

Comment:
Ensure that the `ImageField` with the custom upload path `movie_image_path` is not accessible via POST requests to `api/cinema/movies/`, as per the task requirements. This field should only be used via the `/upload-image/` endpoint.